core/operation/src/test/java/uk/gov/gchq/gaffer/named/operation/AddNamedOperationTest.java

```diff
@@ -94,20 +96,16 @@ public void shouldJsonSerialiseAndDeserialise() {
                 " \"readAccessRoles\" : [ \"User\" ],%n" +
                 " \"writeAccessRoles\" : [ \"User\" ],%n" +
                 " \"readAccessPredicate\" : {%n" +
-                "    \"class\" : \"uk.gov.gchq.gaffer.access.predicate.CustomAccessPredicate\",%n" +
-                "    \"userId\" : \"CreatingUserId\",%n" +
-                "    \"map\" : {%n" +
-                "        \"ReadKey\": \"ReadValue\"%n" +
-                "    },%n" +
-                "    \"auths\" : [ \"CustomReadAuth1\", \"CustomReadAuth2\" ]%n" +
+                "    \"class\" : \"uk.gov.gchq.gaffer.access.predicate.AccessPredicate\",%n" +
+                "    \"userPredicate\" : {%n" +
+                "       \"class\" : \"uk.gov.gchq.gaffer.access.predicate.user.CustomUserPredicate\"%n" +
+                "    }%n" +
                 "},%n" +
                 "\"writeAccessPredicate\" : {%n" +
-                "    \"class\" : \"uk.gov.gchq.gaffer.access.predicate.CustomAccessPredicate\",%n" +
-                "    \"userId\" : \"CreatingUserId\",%n" +
-                "    \"map\" : {%n" +
-                "        \"WriteKey\": \"WriteValue\"%n" +
-                "    },%n" +
-                "    \"auths\" : [ \"CustomWriteAuth1\", \"CustomWriteAuth2\" ]%n" +
+                "    \"class\" : \"uk.gov.gchq.gaffer.access.predicate.AccessPredicate\",%n" +
+                "    \"userPredicate\" : {%n" +
+                "       \"class\" : \"uk.gov.gchq.gaffer.access.predicate.user.CustomUserPredicate\"%n" +
```

Comment:
Where has the predicate gone that is being tested?
```CustomWriteAuth1```

Reply:
This test was not being executed, it's another example of Junit 5 not discovering and running overridden `@Test` methods.  When the `@Test` annotation was added I discovered it was broken. I've removed the `userid`, `map` and `auths` fields as they no longer exist in `uk.gov.gchq.gaffer.access.predicate.CustomAccessPredicate`.